lib/record_list.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'entry'
+
+class RecordList
+
+  attr_reader :list, :balance
+
+  def create_entry(amount, entry = Entry.new)
+    update_balance(amount)
+    entry.set_attributes(amount, @balance)
```

Comment:
Is there a way to inject `Entry` that means you can pass the amount and balance straight into the constructor?